{"uuid":"1bdac3b4-d047-45a8-9bb5-bc2ff8ea747a","historyId":"69221dd516721033b8ee06b9a572902d","fullName":"com.thetestingacademy.ex01_Selenium_Basics.PracticeProjects.Project_2.test_VWO_Login_Invalid_credentials","labels":[{"name":"package","value":"com.thetestingacademy.ex01_Selenium_Basics.PracticeProjects.Project_2"},{"name":"testClass","value":"com.thetestingacademy.ex01_Selenium_Basics.PracticeProjects.Project_2"},{"name":"testMethod","value":"test_VWO_Login_Invalid_credentials"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"LearningSeleniumATB13x"},{"name":"subSuite","value":"com.thetestingacademy.ex01_Selenium_Basics.PracticeProjects.Project_2"},{"name":"host","value":"DVCI6118876"},{"name":"thread","value":"8356@DVCI6118876.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"owner","value":"Mrunal"}],"links":[],"name":"test_VWO_Login_Invalid_credentials","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"current thread is not owner","trace":"java.lang.IllegalMonitorStateException: current thread is not owner\r\n\tat java.base/java.lang.Object.wait0(Native Method)\r\n\tat java.base/java.lang.Object.wait(Object.java:366)\r\n\tat com.thetestingacademy.ex01_Selenium_Basics.PracticeProjects.Project_2.test_VWO_Login_Invalid_credentials(Project_2.java:31)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\r\n\tat org.testng.TestRunner.run(TestRunner.java:616)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","description":"TC#1 _ test invalid login , verify error message","steps":[],"attachments":[],"parameters":[],"start":1756348066755,"stop":1756348092441}